### PR TITLE
Update toolbox

### PR DESCRIPTION
**Pull request by M0hsend**
First go at cleaning this up a bit.
Now by `import epsic_tools.api as epsic` we can get access to all the functionalities.
Removed / merged some of the py files. 
Next to do: 
- [x] Merge what @chrisallen80 has written for ptyREX with mine
- [x] Add tests - Maybe even as test notebooks

### CODE REVIEW

**Comment by TomSlater** [comment]
That took me far too many attempts...

**Comment by M0hsend** [comment]
I have now put everything in ptycho_utils.py and changed the imports accordingly. The example notebook [here](https://github.com/ePSIC-DLS/Staff-notebooks/blob/master/Ptychography_examples/Save_and_Run_PtyRex_json_15kVAu.ipynb) runs with no problems.
These functions can probably be replaced with what I have adde in `get_probe_array(file_path)` and `get_obj_array(file_path)` 
```
	get_hdf5_error(h5_file)
	get_hdf5_object_phase(h5_file, params)
	get_hdf5_object_modulus(h5_file, params)
	get_hdf5_probe_phase(h5_file)
	get_hdf5_probe_modulus(h5_file)
	get_hdf5_complex_probe(h5_file)
	get_json_params(h5_file)
```
But I have not deleted them in case it breaks some of your notebooks @chrisallen80

**Comment by M0hsend** [comment]
Finally managed to get the tests run and pass! (looking over what you have done @TomSlater in ParticleSpy helped a lot 😄). I have written some tests but can definitely expand more in future. Please have a look @TomSlater and @chrisallen80 and comment / merge.

File: epsic_tools/tests/test_ptycho_utils.py [hunk 1]
```diff
@@ -0,0 +1,47 @@
...
+    root_path = os.path.dirname(epsic.__file__)
+    test_json_path = os.path.join(root_path , 'tests', 'test_data', 'test_recon_sim.json')
+    epsic.ptycho_utils.plot_ptyREX_output(test_json_path)
+    
+def test_crop_recon_obj():
+    root_path = os.path.dirname(epsic.__file__)
+    test_json_path = os.path.join(root_path , 'tests', 'test_data', 'test_recon_sim.json')
+    epsic.ptycho_utils.crop_recon_obj(test_json_path)
+  
+def test_get_json_pixelSize():
```
Comment (TomSlater) [inline epsic_tools/tests/test_ptycho_utils.py:16]: Do you want to add assert statements here to test too?

**Comment by TomSlater** [comment]
I think the tests look good. We could add some more assert statements for testing but it's probably good at the moment just to check everything runs. I'll add codecov to the repo so that it's easy to see what isn't being tested too.

**Comment by M0hsend** [comment]
> I think the tests look good. We could add some more assert statements for testing but it's probably good at the moment just to check everything runs. I'll add codecov to the repo so that it's easy to see what isn't being tested too.
> 
> I'm happy to merge at this point if you are.

Great, thanks Tom. Yes, please go ahead and merge.

**Pull request merged**